Sources/MapboxMaps/Gestures/GestureOptions.swift

```diff
@@ -60,5 +60,11 @@ public struct GestureOptions: Equatable {
     /// Defaults to `UIScrollView.DecelerationRate.normal.rawValue`
     public var panDecelerationFactor: CGFloat = UIScrollView.DecelerationRate.normal.rawValue
 
+    /// By default, gestures rotate and zoom around the center of the gesture. Set this property to rotate and zoom around a fixed point instead.
+    ///
+    /// In order to force the map to only revolve around the specified focal point, disable
+    /// ``GestureOptions/pinchPanEnabled`` and ``GestureOptions/panEnabled``. 
```

Comment:
I don't think it's necessary to disable `GestureOptions.panEnabled`

Reply:
Leaving regular pan enabled will enable the visible portion of the map to be moved relative to the specified focal point. So a completely different location might end up under the focal point instead of the location shown initially.

Reply:
@pengdev any thoughts on this?

Reply:
I think if we're going to mention `panEnabled` it should be in the context of keeping a map *coordinate* fixed at specific focal point. That's a slightly different use case than forcing rotation and zoom to happen at a specified point.

Reply:
Made it so 👍